resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -857,7 +858,7 @@ private[spark] class ApplicationMaster(
             finish(FinalApplicationStatus.SUCCEEDED, ApplicationMaster.EXIT_SUCCESS)
           } else {
             logError(log"Driver terminated with exit code ${MDC(EXIT_CODE, exitCode)}! " +
-              log"Shutting down. ${MDC(REMOTE_ADDRESS, remoteAddress)}")
+              log"Shutting down. ${MDC(RPC_ADDRESS, remoteAddress)}")
```

Comment:
This seems unrelated to this PR, but I found that the LogKey `REMOTE_ADDRESS` and `RPC_ADDRESS` have the same meaning, so I removed `REMOTE_ADDRESS` from `LogKey` and only kept `RPC_ADDRESS`